app/models/organization.rb

```diff
@@ -1,2 +1,58 @@
 class Organization < ActiveRecord::Base
+  include Contactable  # this is a placeholder concern for the shared aspects between Organization and People.
+#  include ::Notable  ##JE todo: for next integration pass
+
+  # create_table :organizations do |t|
+  #   ## base Contact fields
+  #   t.references :division, index: true
+  #   t.string :name  # note, also renamed from display_name
+  #   t.string :legal_name
+  #   t.string :primary_phone
+  #   t.string :secondary_phone
+  #   t.string :fax
+  #   t.string :email
+  #   t.text :street_address  # note, this existing data has some verbose directions stored in this field
+  #   t.string :city
+  #   t.string :neighborhood
+  #   t.string :state
+  #   t.references :country
+  #   t.string :tax_no
+  #   t.string :website
+  #   t.text :notes
+  #   t.timestamps
+  #   ## distinct Organization fields
+  #   t.string :alias
+  #   t.references :primary_contact, references: :people
+  #   t.string :sector
+  #   t.string :industry
+  #   t.string :referral_source
+  #   t.references :organization_snapshot
+
+
+  belongs_to :division
+  belongs_to :country
+  belongs_to :primary_contact, class_name: 'Person'
+  belongs_to :organization_snapshot # the latest data
```

Comment:
Doesn't 'has_one' place the key on the other table?  in reality organization 'has_many' snapshots (which is also represented a few lines down)
My intention of the 'belongs_to' is an explicit column on the organization table pointing to the latest record.  This is slightly denormalized as we could do a query fetch one sorted by date.  But if we want to query against the org table by the join, it would be handy to have an easy join column.  This also leads to what kind of sorting and filtering we need to support in the various list views, and how we want to approach that.